app/models/developer.rb

```diff
@@ -23,15 +24,19 @@ class Developer < ApplicationRecord
   validates :bio, presence: true
   validates :time_zone, presence: true, on: :create
   validates :cover_image, content_type: ["image/png", "image/jpeg", "image/jpg"],
-    max_file_size: 10.megabytes
+  max_file_size: 10.megabytes
 
+
+
+ 
   scope :filter_by_utc_offset, ->(utc_offset) { where(utc_offset:) }
   scope :filter_by_role_types, ->(role_types) do
     RoleType::TYPES.filter_map { |type|
       where(role_type: {type => true}) if role_types.include?(type)
     }.reduce(:or).joins(:role_type)
   end
 
+  scope :visible, -> { where(search_status:  [:open, :actively_looking, :not_interested, nil] ) }
```

Comment:
```suggestion
  scope :visible, -> { where.not(search_status:  [:invisible] ) }
```

Reply:
Awesome, Thanks for the feedback. Like I said, I'm not familiar with the way you set up the developers_query class, and Pagy for that matter.  I added the scope to the initialize_pagy_and_developers class. let me know what you think... I added the English and Spanish Locale.
One more thing I might need to do is deny access to developers show view when not authorize developer & invisible